src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -27,21 +27,22 @@
 public class OtpRequestProcessor {
     private static final Logger LOG = LoggerFactory.getLogger(OtpRequestProcessor.class);
     private static final String OTP_SERVER = getConfigPropertyAsText("OTP_SERVER");
-    private static final String OTP_ROOT_ENDPOINT = getConfigPropertyAsText("OTP_ROOT_ENDPOINT");
     private static final String OTP_PLAN_ENDPOINT = getConfigPropertyAsText("OTP_PLAN_ENDPOINT");
 
     /**
      * Register http endpoint with {@link spark.Spark} instance based on the OTP root endpoint. An OTP root endpoint is
      * required to distinguish between OTP and other middleware requests.
      */
     public static void register(Service spark) {
-        // available at (depending on config) http://localhost:4567/otp/*
-        spark.get(OTP_ROOT_ENDPOINT + "/*", OtpRequestProcessor::proxy);
+        // available at http://localhost:4567/otp/routers/default/*
+        spark.get("/otp/routers/default/*", OtpRequestProcessor::proxy);
```

Comment:
Sorry, I think my previous comments were a little vague about this. I think this path should probably just stay `/otp`. For now, I think we want any request to be passed to the API, including requests to the root, which can give information about the version of OTP that is running. E.g., https://fdot-otp-server.ibi-transit.com/otp/

Reply:
Merged #36